doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -478,6 +478,12 @@ This section provides detailed lists of changes by :ref:`script <scripts>`.
 
   * SPDX License List database updated to version 3.18.
 
+* :ref:`partition_manager`:
+
+  * Added:
+
+    * :kconfig:option:`CONFIG_PM_PARTITION_ALIGN_SETTINGS_STORAGE` Kconfig option to specify the alignment of the Settings storage partition.
```

Comment:
Do we need to capitalize the "Settings storage"?

Reply:
Well, it turns out that we don't have to do that:

https://docs.zephyrproject.org/latest/services/settings/index.html

I fixed the capitalization